sonic-chassisd/scripts/chassisd

```diff
@@ -209,6 +224,41 @@ class ModuleUpdater(logger.Logger):
 
         return module_info_dict
 
+    def _is_supervisor(self):
+        if self.my_slot == self.supervisor_slot:
+            return True
+        else:
+            return False
+
+    def check_midplane_reachability(self):
+        if not self.midplane_initialized:
+            return
+
+        index = -1
+        for module in self.chassis.get_all_modules():
+            index += 1
+            #Skip fabric cards
+            if module.get_type() == ModuleBase.MODULE_TYPE_FABRIC:
+                continue
+
+            if self._is_supervisor():
+                #On supervisor skip checking for supervisor
+                if module.get_slot() == self.supervisor_slot:
+                    continue
+            else:
+                #On line-card check only supervisor
+                if module.get_slot() != self.supervisor_slot:
+                    continue
+
+            module_key = try_get(module.get_name, default='MODULE {}'+str(index))
+            midplane_ip = try_get(module.get_midplane_ip, default=INVALID_IP)
+            midplane_access = try_get(module.is_midplane_reachable, default=False)
+
+            #Update db with midplane information
```

Comment:
Add space after `#` per PEP8 standards. Also applies to other locations in the file. Suggest using `autopep8 --max-line-length 120 <filename>` to help files align with the standard.

Reply:
Fixed.